src/tests/test_utils/functional_test_utils/layer_tests_summary/run_parallel.py

```diff
@@ -48,13 +48,18 @@ def parse_arguments():
     working_dir_num_help = "Working dir"
     process_timeout_help = "Process timeout in s"
     parallel_help = "Parallel over HW devices. For example run tests over GPU.0, GPU.1 and etc"
+    split_unit_help = "Split by test or suite"
+    repeat_help = "Number of times to repeat failed and interrupted tests"
 
     parser.add_argument("-e", "--exec_file", help=exec_file_path_help, type=str, required=True)
     parser.add_argument("-c", "--cache_path", help=cache_path_help, type=str, required=False, default="")
     parser.add_argument("-j", "--workers", help=worker_num_help, type=int, required=False, default=(os.cpu_count() - 1) if os.cpu_count() > 2 else 1)
     parser.add_argument("-p", "--parallel_devices", help=parallel_help, type=int, required=False, default=0)
     parser.add_argument("-w", "--working_dir", help=working_dir_num_help, type=str, required=False, default=".")
     parser.add_argument("-t", "--process_timeout", help=process_timeout_help, type=int, required=False, default=DEFAULT_PROCESS_TIMEOUT)
+    parser.add_argument("-s", "--split_unit", help=split_unit_help, type=str, required=False, default="")
+    parser.add_argument("-rf", "--repeat_failed", help=repeat_help, type=int, required=False, default=0)
```

Comment:
`repeat_not_runned`. Please add `-rf=1` to conformance run (product_config ->conf_dldt_tests.py -> generate_conformance_tests)